hkm/static/hkm/css/main.css

```diff
@@ -5890,102 +5875,174 @@ a.btn-delete-row {
     background-color: #fff;
     display: inline-block; }
 
-.basket {
+.basket-row {
+  position: relative;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 30% 50% 10% 10%;
+  height: 410px;
   background-color: #fff;
-  margin: 2em 0; }
-  .basket-row {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    padding: 1.5em;
-    padding-left: 3em;
-    padding-right: 3em;
-    border-bottom: 1px solid rgba(82, 104, 194, 0.05);
-    position: relative; }
-  .basket-footer {
-    background-color: rgba(255, 120, 101, 0.15); }
-  .basket-thumbnail img {
-    max-width: 150px;
-    height: auto;
-    border-radius: 2px;
-    box-shadow: 0 2px 12px -4px rgba(0, 0, 0, 0.5); }
+  margin-bottom: 15px; }
+  @media (min-width: 450px) {
+    .basket-row {
+      height: 300px;
+      grid-template-columns: 175px auto;
+      grid-template-rows: 30% 25% 25%; } }
+  @media (min-width: 850px) {
+    .basket-row {
+      height: 200px;
+      grid-template-columns: 175px 43% 43%;
+      grid-column-gap: 2%;
+      grid-template-rows: 50% 50%; } }
+  @media (min-width: 1050px) {
+    .basket-row {
+      grid-template-columns: 175px 46% 17% 15%;
```

Comment:
I don't know how the layout should look and how it looks now, but this doesn't look very robust to me. It seems like there's an implicit 20% that may or may not be compensated for by the 175px. I would use `fr` so that the grid columns span available space and limit the size of the container with `max-width` to stop them from spanning too much space.

Reply:
Fixed now.